### DISCUSSION TITLE
Shiny with Quarto "Error parsing args at position 0: ..."

### DISCUSSION THREAD

**Discussion by radovan-miletic** [quarto-dev/quarto-cli#879] [Q&A] [answered]
Hi Quarto Dev Team & Quarto Community,  

When running a Shiny [example](https://quarto.org/docs/interactive/shiny/):  

````
---
title: "Old Faithful"
format: html
server: shiny
---

```{r}
sliderInput("bins", "Number of bins:", 
            min = 1, max = 50, value = 30)
plotOutput("distPlot")
```

```{r}
#| context: server
output$distPlot <- renderPlot({
  x <- faithful[, 2]  # Old Faithful Geyser data
  bins <- seq(min(x), max(x), length.out = input$bins + 1)
  hist(x, breaks = bins, col = 'darkgray', border = 'white')
})
```
````  
Immediately an error is thrown:
````
ERROR: TypeError: Error parsing args at position 0: invalid length 0, expected struct StatArgs with 2 elements
````  
I'm wondering if I'm the only one getting this error or has anyone else noticed this happening?

I'm using R-4.2.0 for Windows (2022-04-22 ucrt), Quarto (v0.9 Build 403), RStudio daily build (2022.06.0-daily-405) and {rmarkdown} (v2.14).
Any help or advice is appreciated. Thank you!

**Comment by jjallaire** [comment, marked as answer]
This is a bug we seem to have introduced a couple of weeks ago (thanks for reporting!). It's fixed here (https://github.com/quarto-dev/quarto-cli/commit/1d049e2cc5e7ff429eab40e5669c2ee8d71d9f3a) and available in the updated release v0.9.405 (https://github.com/quarto-dev/quarto-cli/releases/tag/v0.9.405)

**Comment by radovan-miletic** [comment]
Runs smoothly now. Thank you for quick fix!
